Web/Edubase.Web.UI/Areas/Governors/Models/GovernorsGridViewModel.cs

```diff
@@ -130,6 +130,21 @@ public GovernorsGridViewModel()
                     removeGroupEstablishmentSuffix: true
                 );
 
+                // In the roleQuivalence, 'GovernanceProfessionalToAnIndividualAcademyOrFreeSchool' is a title for a list
+                // inside the list is the hard coded string below, and the '...freeSchool'
+                // this code changes the on screen title to display the correct title for the roleID
+
+                if (role == GR.GovernanceProfessionalToAnIndividualAcademyOrFreeSchool)
+                {
+                    foreach (var item in dto.CurrentGovernors)
+                    {
+                        if (item.RoleId == 20)
+                        {
+                            governorRoleNameTitle = "Shared governance professional establishment";
+                        }
```

Comment:
For this, could we use the enums (instead of the int) and naming service (instead of the raw string) to provide these values rather than hard-coding them in?

Reply:
Can change to: if (item.RoleId == (int)GR.Establishment_SharedGovernanceProfessional) if that makes it clearer? - If you mean name change in GovernorRoleNameFactory there isn't a 'Shared governance professional establishment' (there is one with a dash in but that isn't what is expected)